app/assets/stylesheets/application.scss

```diff
@@ -26,3 +26,5 @@
 body[no-nav] nav {
   display: none;
 }
+
+$navbarHeight=71px; /* Obtained from tallest element in navbar+padding+border */
```

Comment:
Think you get the syntax wrong? Should be `$navbarHeight: 71px; // ...`

(yeah you can use `//` comments in SCSS)